plasma_framework/contracts/src/utils/RLPReader.sol

```diff
@@ -197,8 +196,9 @@ library RLPReader {
             // Item is a single byte
             decodedLength = 1;
             offset = 0;
-        } else if (byte0 < STRING_LONG_START) {
-            // Item is a short string (length <= 55 bytes)
+        } else if (STRING_SHORT_START <= byte0 && byte0 < STRING_LONG_START) {
```

Comment:
Nitpick: the added `STRING_SHORT_START <= byte0` (and similar in the other if statements) is unnecessary and costs extra gas.
I guess you put it in to make the code cleaner, I'm in favour of that and am happy to leave it there. Just wanted to point out the extra (probably minimal) gas cost

Reply:
Agree on the extra gas cost, it just drove me mad to jump back and forth 3 times until you are sure again what range byte0 has. I am ok if you want to remove them again